src/Confluent.Kafka/Consumer.cs

```diff
@@ -367,7 +367,7 @@ private void ErrorCallback(IntPtr rk, ErrorCode err, string reason, IntPtr opaqu
         {
             // TODO: Is reason ever different from that returned by err2str?
```

Comment:
Yes. [and in fact @ah- was doing this by deriving from `Handle`]. I didn't like having both `Handle` and `SafeKafkaHandle` + some other reasons for changing away from this. This duplicate functionality could all be put in SafeKafkaHandle easily enough and passed through.